tools/rnasum/0.4.7/rnasum__0.4.7.cwl

```diff
@@ -0,0 +1,307 @@
+cwlVersion: v1.1
+class: CommandLineTool
+
+# Extensions
+$namespaces:
+    s: https://schema.org/
+    ilmn-tes: http://platform.illumina.com/rdf/ica/
```

Comment:
Should be `ilmn-tes: https://platform.illumina.com/rdf/ica/`

Reply:
updated in [74f092d](https://github.com/umccr/cwl-ica/pull/350/commits/74f092dd51cb782f538c1ca240df2ab8c4c6354e)